### DISCUSSION TITLE
cors missing allow origin

### DISCUSSION THREAD

**Discussion by binarytide** [supabase/supabase#2550] [Questions]
I'm trying to use gotrue docker behind nginx proxy manager and it fails prefetch with "cors missing allow origin" unless I disable cors via cors everywhere plugin.

I've set `API_EXTERNAL_URL`, `GOTRUE_URI_ALLOW_LIST`, and `GOTRUE_JWT_AUD` but still no luck

**Comment by inian** [comment]
If you are using NGINX in front of gotrue, can configure nginx to send the required CORS headers @binarytide? Example [here](https://serverfault.com/questions/162429/how-do-i-add-access-control-allow-origin-in-nginx)
